Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM apache/airflow:2.9.0
+FROM apache/airflow:2.10.3
```

Comment:
Do we want to consider pinning to minor release instead of patch release? so `apache/airflow:2.10`  instead of `apache/airflow:2.10.3`?

Edit: I see that the Airflow docker repository does not follow the tagging convention needed to enable this :/